client/galaxy/style/less/base.less

```diff
@@ -1719,3 +1719,23 @@ div.toolTitleNoSection
 #for_bears {
     display: none;
 }
+
+// communication channel bootstrap modal 
```

Comment:
@bgruening Can we create a separate folder in the config/plugins directory for the communication channel codebase and move files like generic-nav-view.js and the communication folder? 
For this `+// communication channel bootstrap modal `
we can create a new css file and move these styles there. It cannot go within the communication.css file as it loads inside the iframe.